components/segmenter_lstm/tests/testdata/test_text_codepoints.json

```diff
@@ -0,0 +1,3 @@
+{
+  "testcases": [{"unseg": "ข้อหาแล้ว", "true_bies": "biebebiie"}, {"unseg": "ไปยังเพื่อผัดฟ้องฝากขังตามกำหนด", "true_bies": "bebiebiiiebiebiiebiebiebiebiiie"}, {"unseg": "สำหรับผู้ต้องหาอีก", "true_bies": "biiiiebiebiiebebie"}]
```

Comment:
Add at least 20, maybe 100 (here and below in graphclust)

Reply:
Manually added 30 for each of codepoints and graphclsut